package.json

```diff
@@ -122,10 +122,10 @@
   },
   "devDependencies": {
     "@apollo/react-testing": "^3.1.4",
-    "@graphql-codegen/cli": "^1.13.3",
-    "@graphql-codegen/typescript": "^1.13.3",
-    "@graphql-codegen/typescript-operations": "^1.13.3",
-    "@graphql-codegen/typescript-react-apollo": "^1.13.3",
+    "@graphql-codegen/cli": "1.13.3",
+    "@graphql-codegen/typescript": "1.13.3",
+    "@graphql-codegen/typescript-operations": "1.13.3",
+    "@graphql-codegen/typescript-react-apollo": "1.13.3",
```

Comment:
If i add "^" back, it installs wrong version and codegen won't work with current script

Reply:
Ok, Better to use strict version in that case. Maybe all codegen related libraries could be updated at some point.

Reply:
yeah